website/pages/en/cookbook/enums.mdx

```diff
@@ -0,0 +1,245 @@
+---
+title: Categorize NFT Marketplaces Using Enums
+---
+
+Learn how to effectively organize information from transactions and marketplace interactions using Enums.
+
+> Note: This guide uses the CryptoCoven NFT smart contract.
+
+## What are Enums?
+
+Enums, or enumeration types, are a specific data type that allows you to define a set of allowed values.
```

Comment:
I find an explanation without an example hard to follow. 

I suggest throwing in an example immediately before moving on to a new section. Perhaps no need for the `Enums Symtax` header below. Instead:
-Explain enum
-Show example right away
-Further explanations (the other content from Enums Syntax).